### PR TITLE
Suspend reconcile

### PR DESCRIPTION
**Pull request by kerenlahav**

### CODE REVIEW

File: controllers/servicebinding_controller.go [hunk 1]
```diff
@@ -66,6 +66,12 @@ func (r *ServiceBindingReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	}
 	serviceBinding = serviceBinding.DeepCopy()
 
+	if r.Config.SuspendReconcile {
+		log.Info("operator is suspended")
+		setBlockedCondition("operator is suspended", serviceBinding)
+		return ctrl.Result{}, nil
```
Comment (pavelmaliy) [inline controllers/servicebinding_controller.go:72]: maybe check if no conditions do the setBlockedCondition

Comment (pavelmaliy) [reply controllers/servicebinding_controller.go:72]: need to update the status no?

File: controllers/serviceinstance_controller.go [hunk 2]
```diff
@@ -98,11 +104,10 @@ func (r *ServiceInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Requ
...
-			setFailureConditions(smTypes.CREATE, err.Error(), serviceInstance)
-			if err := r.updateStatusWithRetries(ctx, serviceInstance, log); err != nil {
-				return ctrl.Result{}, err
+			if isTransientError(err, log) {
+				return r.markAsTransientError(ctx, smTypes.CREATE, err.Error(), serviceInstance, log)
 			}
-			return ctrl.Result{Requeue: true, RequeueAfter: r.Config.SyncPeriod}, nil
+			return r.markAsNonTransientError(ctx, smTypes.CREATE, err.Error(), serviceInstance, log)
 		}
 		if instance != nil {
```
Comment (pavelmaliy) [inline controllers/serviceinstance_controller.go:112]: it can be update as well, but I think we have no way to know it

Comment (kerenlahav) [reply controllers/serviceinstance_controller.go:116]: what do you mean? 
if the error is non-transient there is no need to requeue

**Pull request closed without merging**